### PR TITLE
Add warning section in database migration

### PR DESCRIPTION
**Pull request by gaelreyrol**
Add a warning section for database migration to warn users that we don't keep read only users

### CODE REVIEW

**Comment by KannarFr** [comment]
@gaelreyrol Thanks for the PR.

@ldoguin WDYT to put it before the migrations information to ensure users see it?

**Comment by ldoguin** [comment]
@KannarFr yes, also are there other limitations ? I am assuming anything custom has to be done again.

**Comment by KannarFr** [comment]
@gaelreyrol can you change the content order? 

@ldoguin nothing yet, but probably, will make PR if required.

**Comment by gaelreyrol** [comment]
@KannarFr Done ! I also used a warning alert instead

**Comment by KannarFr** [comment]
@ldoguin can review & possibly merge it?

File: administrate/database-migration.md [hunk 1]
```diff
@@ -11,21 +11,30 @@ tags:
 
 ## Migrating any database
 
+{{< alert "warning" "Read only users" >}}
+    As we do not support the migration of read only users, if you have created read only users you will have to create new ones.
```
Comment (BlackYoup) [inline administrate/database-migration.md:15]:
```suggestion
    We do not support the migration of read only users. If you have any, you will have to create new ones at the end of the migration.
```

**Comment by BlackYoup** [comment]
Thanks!

**Pull request merged**